installer/scripts/service_control

```diff
@@ -222,11 +490,11 @@ find_systemd_dir()
     # Various distributions have different paths for systemd unit files ...
     local UNIT_DIR_LIST="/usr/lib/systemd/system /lib/systemd/system"
 
-    if pidof systemd 1> /dev/null 2> /dev/null; then
+    if pidof systemd 1> /dev/null 2>&1; then
```

Comment:
Why have you changed this line? If the system does not have systemd, it should have something else, and that's expected.

Reply:
I think I have all those now.  Pushing any second.

Reply:
Please answer my question here; this line is still changed.

Reply:
I don't understand the question.  The intent is to use 2>&1; instead of the duplicated form.  I apologize if there is something else I am not seeing.  I'll come over.